confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/internal/macros/AbstractPageMacroConverter.java

```diff
@@ -0,0 +1,5 @@
+package org.xwiki.contrib.confluence.filter.internal.macros;
+
+public class AbstractPageMacroConverter
```

Comment:
Looks like something committed by mistake (at least I don't see the point of that class).